docs/to_clarify.md

```diff
@@ -18,16 +18,26 @@
 
 1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
 
-    a. What needs to be done during the checking-in process?
+    a. What needs to be done during the digital checking-in process? Are there any particular details/ workflow where it must be adhered to provide a check-in process?
 
     b. Which part of the process can guests complete up to before arrival, and what else do they need to do to complete the check-in process upon arrival?
 
     c. On the same note, what needs to be done during the checking-out process?
 
+    d. I would like to confirm the current traditional workflow of the check-in process.(Please fill me in if i got the procedure wrong)
+    Guest - Provide personal particulars, Photos of Passport/Identity Card, Booking Documents.?
+    
+    Front Desk staffs - Receiving the guest and collect relevant documents from guest. Do verification and give out hotels access card. Guide Guest to their room.
+
+
```

Comment:
Unnecessary new line

Reply:
i see that it is not relevant anymore. That's why i didnt add in.

Reply:
I think your above message is written in response to my comment https://github.com/openinnovation-hotel-check-in-out/ideation/pull/9#pullrequestreview-459692826. In that case, you shouldn't reply on this thread. Rather, you should reply by creating a new comment below and quoting the relevant segments.